test/one_dragon/envs/test_git_service.py

```
@@ -106,6 +108,67 @@ def test_remove_temp_repo_handles_readonly_pack_files(tmp_path: Path) -> None:
+    monkeypatch.setattr(git_service_module.sys, 'platform', 'win32')
+    monkeypatch.setattr(git_service_module.shutil, 'rmtree', lambda *args, **kwargs: (_ for _ in ()).throw(error))
+    monkeypatch.setattr(git_service_module.log, 'info', info_messages.append)
+    monkeypatch.setattr(
+        git_service_module.log,
+        'warning',
+        lambda message, **kwargs: warning_messages.append(message),
+    )
+
+    _remove_temp_repo('fetch_busy')
+
+    assert info_messages == ['Git fetch 临时仓库仍被占用，将在下次启动时清理: fetch_busy']
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)